src/main/java/edu/kis/powp/jobs2d/command/visitor/CommandVisitorCounter.java

```diff
@@ -0,0 +1,36 @@
+package edu.kis.powp.jobs2d.command.visitor;
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.ICompoundCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+import java.util.Iterator;
+
+public class CommandVisitorCounter implements CommandVisitorInterface {
+
+    private int operateToCommandCounter = 0;
+    private int setPositionCommandCounter = 0;
```

Comment:
Why counting different kinds of commands if we cannot access and use specific information.